Assets/Scripts/Layers/LayerTypes/HierarchicalObjectLayerGameObject.cs

```diff
@@ -8,14 +10,26 @@
 
 namespace Netherlands3D.Twin.Layers
 {
-    public class HierarchicalObjectLayerGameObject : LayerGameObject, IPointerClickHandler, ILayerWithProperties
+    public class HierarchicalObjectLayerGameObject : LayerGameObject, IPointerClickHandler, ILayerWithPropertyPanels, ILayerWithPropertyData
     {
         private ToggleScatterPropertySectionInstantiator toggleScatterPropertySectionInstantiator;
         [SerializeField] private UnityEvent<GameObject> objectCreated = new();
         private List<IPropertySectionInstantiator> propertySections = new();
+        private TransformLayerPropertyData transformPropertyData;
+        private Vector3 previousPosition;
+        private Quaternion previousRotation;
+        private Vector3 previousScale;
+
+        LayerPropertyData ILayerWithPropertyData.PropertyData => transformPropertyData;
 
         protected void Awake()
         {
+            var coord = new Coordinate(CoordinateSystem.Unity, transform.position.x, transform.position.y, transform.position.z);
+            transformPropertyData = new TransformLayerPropertyData(coord, transform.eulerAngles, transform.localScale);
+            // transformPropertyData.Position = new Coordinate(CoordinateSystem.Unity, transform.position.x, transform.position.y, transform.position.z);
```

Comment:
Moet deze commented code herstelt of verwijdert worden?

Reply:
weer over het hoofd gezien, thanks